### ISSUE TITLE
OSARA doesn't support ignore project tempo 

### ISSUE THREAD

**Issue by RDMurray** [jcsteh/osara#305]
If ignore project tempo is checked in item source properties, and the item tempo differs from the project, OSARA gets note positions and lengths wrong.  I tried to fix this, but gave up for now due to the API not providing all the information necessary.

If we depend on sws for this, the only thing still missing is the ticks per quarter note for the take.  I tried to calculate it with MIDI_GetPPQPos_StartOfMeasure() and MIDI_GetPPQPos_EndOfMeasure() but those functions don't work the way I expected them to.  MIDI_GetPPQPos_EndOfMeasure seems to just return the position you pass to it.

**Comment by ScottChesworth** [comment]
I've emailed Cockos about the unexpected results @RDMurray is getting. Will report back as soon as I hear back.

**Comment by ScottChesworth** [comment]
Apologies @RDMurray , I seem to have lost track of this issue in the carcrash of my email. I remember passing a response from Cockos along to you about something, but can't remember whether this was what it pertained to. If it was, did that response help any, or do I need to press harder?

**Comment by RDMurray** [comment]
No, I didn't hear anything about this but  I think it is probably low priority due to it being a rarely used feature.
